.github/workflows/ci-cd.yml

```diff
@@ -30,58 +37,292 @@ jobs:
         with:
             url: "https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe"
             target: public/
-                
+
       - name: Install targeting pack
         shell: cmd
         working-directory: public
         run: NDP461-DevPack-KB3105179-ENU.exe /q
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v2
-        
+
+      # We need the dotnet runtime to execute overcrowdin
       - name: Setup dotnet
-        uses: actions/setup-dotnet@v3
+        uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: | 
-            2.1.x
-            3.1.x
-            5.0.x
+          dotnet-version:
+            8.0.x
         if: github.event_name != 'pull_request'
-        
+
+      - name: Downgrade Wix Toolset - remove when runner has 3.14.2
```

Comment:
```suggestion
      - name: Downgrade Wix Toolset - remove when runner has 3.14.2  See: https://github.com/actions/runner-images/issues/9667
```

Reply:
I'd rather put that in a comment rather than make an already-long step name even longer.

Reply:
good call.  I moved it to the next line as a comment